docs/DeveloperGuide.md

```diff
@@ -659,6 +659,52 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 <hr>
 
+**System**: FitBook
+
+**Use case**: UC05 - Adding/removing weight of a client
+
+**Actor**: User
+
+**MSS**
+
+1. User requests to see a list of all clients.
+1. FitBook displays a success message followed by the list of clients.
+1. User views the list and adds/deletes a weight value of a client.
+1. FitBook displays a success message indicating that the weight of a client has been modified successfully.
+1. Use case ends
+
+**Extensions**
+
+* 3a. User enters an invalid command.
+    * 3a1. FitBook alerts the user that the command is invalid and displays the correct format.
+    * Use case ends.
+* 3b. User wants to remove a weight value.
+    * 3b1. If there are weight values to be removed, FitBook displays a success message. Else, FitBook displays an error message indicating that there are no weight values to be removed.
```

Comment:
Might want to indicate "latest weight" or something similar? Currently implying that more than one weight value will be removed.